src/js/types/networks/cardano.js

```diff
@@ -19,42 +19,90 @@ export type CardanoPublicKey = {
 
 // GetAddress
 
-export type CardanoGetAddress = {
+export const CARDANO_ADDRESS_TYPE = Object.freeze({
```

Comment:
this file should contains types only

pls move this const to `js/constants/cardano`

you can also put PROTOCOL_MAGICS and NETWORK_IDS there

Reply:
Moved [here](https://github.com/trezor/connect/pull/638/commits/bd67fef49aa1297383e7500cd07f567ce1c327eb).